lib/formatters/stylish.js

```diff
@@ -21,6 +21,16 @@ function pluralize(word, count) {
     return (count === 1 ? word : `${word}s`);
 }
 
+/**
+ * Returns true if and only if any fixes were applied to any input files
+ * @param {Object[]} results - Collection of messages from all the files
+ * @returns {boolean} If any fixes were applied
+ * @private
+ */
+function fixesWereApplied(results) {
+    return Boolean(results && results.some(r => r.output));
```

Comment:
Is `results && ...` necessary here? From what I can tell, `results` will always be truthy.